packages/spartan/extra/dcos-spartan.service

```diff
@@ -19,6 +19,6 @@ ExecStartPre=/usr/bin/env ip link set spartan up
 ExecStartPre=-/usr/bin/env ip addr add 198.51.100.1/32 dev spartan
 ExecStartPre=-/usr/bin/env ip addr add 198.51.100.2/32 dev spartan
 ExecStartPre=-/usr/bin/env ip addr add 198.51.100.3/32 dev spartan
-ExecStart=/opt/mesosphere/active/spartan/spartan/bin/spartan-env foreground
+ExecStart=/usr/bin/env flock -x /opt/mesosphere/active/spartan/spartan/bin/spartan-env /opt/mesosphere/active/spartan/spartan/bin/spartan-env foreground
```

Comment:
@cmaloney I could have a one-liner which is a wrapper for spartan-env -- would that be preferrable?